orchagent/saihelper.cpp

```diff
@@ -200,18 +200,6 @@ void initSaiRedis(const string &record_location)
         }
     }
 
-    /* Disable/enable SwSS recording */
```

Comment:
why remove this part?

Reply:
It duplicates with https://github.com/Azure/sonic-swss/blob/v1.0.3/orchagent/main.cpp#L164,  and causes file write issue.

Reply:
ah! I see. thanks! Let me fix this in a separate pull request!

Reply:
https://github.com/Azure/sonic-swss/pull/298 thanks.

Reply:
https://github.com/Azure/sonic-swss/pull/299